src/pages/billing/List/BillingProjectActions.ts

```diff
@@ -17,38 +19,39 @@ interface BillingProjectActionsProps {
 export const BillingProjectActions = (props: BillingProjectActionsProps) => {
```

Comment:
We have gone from a menu to a single button (the menu only ever had 1 item, and Lou agreed that this was better UX). I'm somewhat tempted to rename the file….